docs/developer/architecture.rst

```diff
@@ -157,42 +106,10 @@ be better optimized for a given execution can be overridden and optimized in tha
 
 This layer has a significantly reduced API from the QueryCompiler and the user-facing
 API. Each of these APIs represents a single way of performing a given operation or
-behavior. Some of these are expanded for convenience/understanding. The API abstractions
-are as follows:
+behavior. Some of these are expanded for convenience/understanding.
```

Comment:
It says they are expanded on, but I'm not sure I see where the expanded description is?

Reply:
I think we can delete this sentence since we now say the following below:
```
More documentation can be found internally in the code_. This API is not complete, but
represents an overwhelming majority of operations and behaviors.
```